dear_petition/petition/api/viewsets.py

```diff
@@ -343,6 +343,15 @@ def combine_batches(self, request):
         label = request.data['label']
         user_id = request.user.id
 
+        if not batch_ids:
```

Comment:
I'm guessing this one was supposed to be for missing label (they are both batch_ids).